tests/Unit/ApiTest.php

```
@@ -715,4 +715,63 @@ public function test_read_feeds_creates_read_feeds_request() {
 			$response->data
 		);
 	}
+
```

Can you add a test to check unhappy path
If errors are thrown or logs are generated

---

I think it won't bring value in this case, as we just mock response. Test with error mocks were added for other calls. that should be enough I think.